common/util/goal_distance_calculator/Readme.md

```diff
@@ -0,0 +1,43 @@
+# autoware_state_monitor
+
+## Purpose
+
+This node publishes deviation of self-pose from goal pose.
+
+## Inner-workings / Algorithms
+
+## Inputs / Outputs
+
+### Input
+
+| Name                               | Type                                      | Description           |
+| ---------------------------------- | ----------------------------------------- | --------------------- |
+| `/planning/mission_planning/route` | `autoware_auto_planning_msgs::msg::Route` | Used to get goal pose |
+| `/tf`                              | `tf2_msgs/TFMessage`                      | TF (self-pose)        |
+
+### Output
+
+| Name                     | Type                                       | Description                                                   |
+| ------------------------ | ------------------------------------------ | ------------------------------------------------------------- |
+| `deviation/lateral`      | `autoware_debug_msgs::msg::Float64Stamped` | publish lateral deviation of self-pose from goal pose[m]      |
+| `deviation/longitudinal` | `autoware_debug_msgs::msg::Float64Stamped` | publish longitudinal deviation of self-pose from goal pose[m] |
+| `deviation/yaw`          | `autoware_debug_msgs::msg::Float64Stamped` | publish yaw deviation of self-pose from goal pose[rad]        |
+| `deviation/yaw_deg`      | `autoware_debug_msgs::msg::Float64Stamped` | publish yaw deviation of self-pose from goal pose[deg]        |
```

Comment:
```suggestion
| Name                     | Type                                       | Description                                                    |
| ------------------------ | ------------------------------------------ | -------------------------------------------------------------- |
| `deviation/lateral`      | `autoware_debug_msgs::msg::Float64Stamped` | Publish lateral deviation of self-pose from goal pose [m]      |
| `deviation/longitudinal` | `autoware_debug_msgs::msg::Float64Stamped` | Publish longitudinal deviation of self-pose from goal pose [m] |
| `deviation/yaw`          | `autoware_debug_msgs::msg::Float64Stamped` | Publish yaw deviation of self-pose from goal pose [rad]        |
| `deviation/yaw_deg`      | `autoware_debug_msgs::msg::Float64Stamped` | Publish yaw deviation of self-pose from goal pose [deg]        |
```
Nit: Might be better to insert a space before the unit, and use upper case letters as same as the other tables.